test/parallel/test-https-truncate.js

```diff
@@ -1,45 +1,45 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-var https = require('https');
+const https = require('https');
 
-var fs = require('fs');
+const fs = require('fs');
 
-var key = fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem');
-var cert = fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem');
+const key = fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem');
+const cert = fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem');
 
 // number of bytes discovered empirically to trigger the bug
-var data = Buffer.allocUnsafe(1024 * 32 + 1);
+const data = Buffer.allocUnsafe(1024 * 32 + 1);
 
 httpsTest();
 
 function httpsTest() {
-  var sopt = { key: key, cert: cert };
+  const sopt = { key: key, cert: cert };
 
-  var server = https.createServer(sopt, function(req, res) {
+  const server = https.createServer(sopt, common.mustCall(function(req, res) {
     res.setHeader('content-length', data.length);
     res.end(data);
     server.close();
-  });
+  }));
 
   server.listen(0, function() {
-    var opts = { port: this.address().port, rejectUnauthorized: false };
-    https.get(opts).on('response', function(res) {
+    const opts = { port: this.address().port, rejectUnauthorized: false };
+    https.get(opts).on('response', common.mustCall(function(res) {
       test(res);
-    });
+    }));
   });
 }
 
 
 function test(res) {
   res.on('end', function() {
```

Comment:
There is still no guarantee that this will be called. You'd need to add `common.mustCall()` here and to `server.listen(0, ...)`.